src/TableHeaderCells.js

```diff
@@ -0,0 +1,67 @@
+import _ from "lodash"
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import styled from 'styled-components'
+
+const TableSearchHeaderCellDiv = styled.div`
+ display: ruby;
+`
+
+function renderTableSortHeaderCell (columnNumber, headerText, width, orderedColumn, ordering, onClick) {
+  return <Table.TextHeaderCell
+    key={`header${columnNumber}`}
+    className={`header${columnNumber}`}
+    flexBasis={width} flexShrink={100} flexGrow={100}
+    onClick={() => onClick(columnNumber)}>
+    <TableSearchHeaderCellDiv>
+      {[
+        `${headerText} `,
+        columnNumber===orderedColumn ?
+          ordering ?
+            <i key={`up${columnNumber}`} className={`icon icon-common icon-sort-up`}/> : <i key={`down${columnNumber}`} className={`icon icon-common icon-sort-down`}/>
+          : <i key={`updown${columnNumber}`} className={`icon icon-common icon-sort`}/>
+      ]}
+    </TableSearchHeaderCellDiv>
+  </Table.TextHeaderCell>
+
+}
+
+function renderTableSearchHeaderCell(columnNumber, headerText, width, searchedColumn, searchQuery, onChange){
+  return <Table.SearchHeaderCell
+    key={`searchheader${columnNumber}`}
```

Comment:
You're not really meant to put keys when it is not obvious the element is part of an array. You should add the key at the point of calling the function. Again, read the React documentation on incorrect key usage: https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys

Perhaps it would be clearer if this `renderTableSearchHeaderCell` was actually a component? Same comment applies to `renderTableSortHeaderCell`.